modules/learn/pages/data/validity.adoc

```diff
@@ -0,0 +1,70 @@
+= Validity Guarantees (ACID)
+
+[abstract]
+Couchbase Server validates data by means of _Atomicity_, _Consistency_, _Isolation_, and _Durability_.
```

Comment:
Is 'validates' the right term?  That brings JSON schema validation to mind, for me.

Reply:
Well, I based it on Wikipedia, which says that ACID is 'a set of properties of database transactions intended to guarantee validity even in the event of errors, power failures, etc'. We can change to something better, but I think we need to explain what binds these four properties together.